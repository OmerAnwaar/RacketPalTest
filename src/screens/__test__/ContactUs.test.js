import React from 'react';
import renderer from 'react-test-renderer';
import ContactUs from '../ContactUs';

describe('Contact Screen', () => {
  it('renders correctly', () => {
    const snapshot = renderer.create(<ContactUs />).toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});
