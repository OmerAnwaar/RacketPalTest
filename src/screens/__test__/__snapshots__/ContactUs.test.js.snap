// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Screen renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    ContactUs
  </Text>
</View>
`;
